### PR TITLE
Added license header

### PR DESCRIPTION
**Pull request by dutterbutter**
## Changes
- Added license header to source files
- 
- 

## Tests:
```

```

### Fixes:

### CODE REVIEW

**Comment by noot** [comment]
@dutterbutter why are the headers double commented ie `// //` instead of `//`

**Comment by dutterbutter** [comment]
> @dutterbutter why are the headers double commented ie `// //` instead of `//`

It's just how the script inputted them as. I used https://github.com/google/addlicense but with the GNU license.

**Comment by ansermino** [comment]
This is not the correct output :cry:  https://github.com/google/addlicense/blob/master/testdata/expected/file.go

**Comment by dutterbutter** [comment]
> This is not the correct output 😢 https://github.com/google/addlicense/blob/master/testdata/expected/file.go

That is for apache license, I specified file explicitly using `-f`

**Comment by noot** [comment]
/@dutterbutter does your input file commented already? if so it probably shouldn't be

**Comment by dutterbutter** [comment]
> @dutterbutter is your input file commented already? if so it probably shouldn't be

Yes that what it is, I will change and re-push

**Comment by dutterbutter** [comment]
@noot @ansermino double comments have been removed

File: rpc/json2/server_test.go [hunk 1]
```diff
@@ -1,3 +1,19 @@
+// // Copyright 2019 ChainSafe Systems (ON) Corp.
```
Comment (ansermino) [inline rpc/json2/server_test.go]: Doubled

**Comment by dutterbutter** [comment]
@noot @ansermino fixed double comments and added to new trie files

**Pull request merged**